Doc/using/android.rst

```diff
@@ -0,0 +1,65 @@
+.. _using-android:
+
+=======================
+Using Python on Android
+=======================
+
+Python on Android is unlike Python on desktop platforms. On a desktop platform,
+Python is generally installed as a system resource that can be used by any user
+of that computer. Users then interact with Python by running a :program:`python`
+executable and entering commands at an interactive prompt, or by running a
+Python script.
+
+On Android, there is no concept of installing as a system resource. The only unit
+of software distribution is an "app". There is also no console where you could
+run a :program:`python` executable, or interact with a Python REPL.
+
+As a result, the only way you can use Python on Android is in embedded mode --- that
```

Comment:
```suggestion
As a result, the only way you can use Python on Android is in embedded mode -- that
```

Reply:
I tried this, but the double-hyphen was rendered as a single hyphen rather than an en dash. Unless anyone knows how to fix this, I'll probably just replace it with a Unicode en dash.

Reply:
Done.